### ISSUE TITLE
6.0.8 EpplusTableColumnAttribute NumberFormat is ignored when Header contains space

### ISSUE THREAD

**Issue by recumbented** [EPPlusSoftware/EPPlus#724]
EpplusTableColumnAttribute.NumberFormat is ignored when Header contains space on 6.0.8.

```cs
void Main()
{
	ExcelPackage.LicenseContext = LicenseContext.NonCommercial;

	var data = new[] { new TableItem() };
	using var excel = new ExcelPackage(File.Create($@"{System.Reflection.Assembly.GetAssembly(typeof(ExcelPackage)).GetName().Version}.xlsx"));
	var sheet = excel.Workbook.Worksheets.Add("sheet");
	sheet.Cells[1, 1].LoadFromCollection(data);
	excel.Save();
}
[EpplusTable(PrintHeaders = true, AutofitColumns = true)]
public class TableItem
{
	[EpplusTableColumn(Header = "EPPlusVersion")]
	public String EPPlusVersion => System.Reflection.Assembly.GetAssembly(typeof(ExcelPackage)).GetName().Version.ToString();

	[EpplusTableColumn(Header = "DateTime", NumberFormat = "yyyy/mm/dd")]
	public DateTime DateTime1 => new DateTime(2022, 09, 30);

	[EpplusTableColumn(Header = "Date Time", NumberFormat = "yyyy/mm/dd")]
	public DateTime DateTime2 => new DateTime(2022, 09, 30);
}
```
![2022-09-30_17h25_36](https://user-images.githubusercontent.com/22209921/193227041-918bcf11-f817-4c7e-b9a6-6e5a915d362b.png)

**Comment by swmal** [comment]
Thanks for reporting, we'll have a look at this.

**Comment by swmal** [comment]
Many thanks for a very good description of this error. I have fixed it in our development branch - you can test the fix via our dev nuget feed, see this wiki page: https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed

Will be included in the next version.

**Comment by cearny** [comment]
In the meantime, will downgrading to 6.0.7 fix this?

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.1
